Eastern-Research-Group/mywaterway
# Moved away from using deprecated Helmet nocache middleware

## Related Issues:
* N/A

## Main Changes:
* Removed the use of the deprecating Helment nocache middleware (https://helmetjs.github.io/docs/nocache/)
 
## Steps To Test:
1. Run the app
2. inspect any of the mywaterway domain provided files and make sure the following headers are present

'Surrogate-Control', 'no-store'
'Cache-Control','no-store, no-cache, must-revalidate, proxy-revalidate'
'Pragma', 'no-cache'
'Expires', '0'
